src/algoliasearch.helper.js

```diff
@@ -22,60 +22,60 @@ function AlgoliaSearchHelper( client, index, options ) {
 
 util.inherits( AlgoliaSearchHelper, events.EventEmitter );
 
+/**
+ * Start the search with the parameters set in the state.
+ * @return {AlgoliaSearchHelper}
+ */
+AlgoliaSearchHelper.prototype.search = function(){
+  this._search();
+  return this;
+};
 
 /**
- * Perform a query
+ * Sets the query. Also sets the current page to 0.
  * @param  {string} q the user query
- * @param {boolean} holdSearch Optionnal parameter to specify that the search should not be triggered right away
+ * @return {AlgoliaSearchHelper}
  */
-AlgoliaSearchHelper.prototype.search = function( q, holdSearch ) {
+AlgoliaSearchHelper.prototype.setQuery = function( q ) {
   this.state = this.state.setQuery( q )
                          .setPage( 0 );
 
```

Comment:
Are we really returning the state here? I assume we're returning `this`, isn't it?